workflow-trace-viewer/src/jvmMain/kotlin/com/squareup/workflow1/traceviewer/util/JsonParser.kt

```diff
@@ -8,8 +8,15 @@ import com.squareup.workflow1.traceviewer.model.Node
 import io.github.vinceglb.filekit.PlatformFile
 import io.github.vinceglb.filekit.readString
 
+/*
+ The root workflow Node uses an ID of 0, and since we are filtering childrenByParent by the
+ parentId, the root node has a parent of -1 ID. This is reflected seen inside android-register
+ */
+const val ROOT_ID: String = "-1"
```

Comment:
My guess is when the parent is null in the `WorkflowSession` you have just given it a default value of -1 in your recording code?